docs/MO_DG/prepare_model/Config_Model_Optimizer.md

```diff
@@ -124,19 +116,15 @@ virtualenv -p /usr/bin/python3.6 .env3/bin/activate
 ```shell
 pip3 install -r requirements.txt
 ```
-> **NOTE**: TensorFlow 1.x and 2.x are incompatible. Use separate virtual environments if you want to install multiple TensorFlow versions.
+> **NOTE**: Since OpenVINO 2022.1 release TensorFlow1 usage is deprecated. TensorFlow 1.x and 2.x are incompatible. Use separate virtual environments if you want to install multiple TensorFlow versions.
 
```

Comment:
please correct the note. Now it is incorrect and absolutely confusing:
NOTE: Support of MO in TensorFlow 1.x environment is deprecated. Due to this `requirements_tf.txt` dependency is changed to TensorFlow 2.x. Use TensorFlow 2.x environment to convert both TensorFlow 1.x and 2.x models.

Reply:
Note updated.